src/styles/main.scss

```diff
@@ -0,0 +1,4 @@
+@import '/src/styles/utils/variables';
+@import '/src/styles/blocks/page';
+@import '/src/styles/blocks/card';
+@import '/src/styles/blocks/stars';
```

Comment:
You are already in the `/src/styles` folder, could you simplify pathes like this `@import '/utils/variables';`?